### ISSUE TITLE
`React-Hot-Loader: Hot Module Replacement is not enabled` when running tests in `jest`

### ISSUE THREAD

**Issue by nryoung** [gaearon/react-hot-loader#1259]
### Description

React Hot Loader is reporting the following error when using the `react-hot-loader/babel` plugin and attempting to run tests with `jest`.

```
 PASS  src/App.test.jsx
  App
    ✓ renders (10ms)

  console.error node_modules/react-hot-loader/index.js:9
    React-Hot-Loader: Hot Module Replacement is not enabled
```

### Expected behavior

When using the babel plugin it should not report this error when executing in the test context.

### Actual behavior

See error above, it is still attempting to load `react-hot-loader` when running tests. The reason I know this is the babel plugin is because the error goes away when I remove the plugin from `.babelrc`.

### Environment

React Hot Loader version: `4.8.3`

Run these commands in the project folder and fill in their results:

1.  `node -v`: `10.15.3`
2.  `yarn -v`: `1.15.2

Then, specify:

1.  Operating system: `OS X 10.14.4`

## App.jsx
```javascript
import React from 'react';
import { hot } from 'react-hot-loader/root';

const App = () => (
 <div>Here is my app</div>
);

export default hot(App);
```

### App.test.jsx
```javascript
import React from 'react';
import { shallow } from 'enzyme';

import App from './App';

describe('App', () => {
  it('renders', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
```

### .babelrc
```
{
    "presets": ["@babel/preset-env", "@babel/preset-react"],
    "plugins": ["react-hot-loader/babel"]
}
```

### package.json
```
{
  "scripts": {
    "test": "jest --color"
  }
}
```

### jest.config.js

```javascript
/* eslint-disable max-len */
process.env.BABEL_ENV = 'test'; // Make sure to load the appropriate babel plugins
process.env.NODE_ENV = 'test'; // set the proper node env

module.exports = {
  setupFilesAfterEnv: ['<rootDir>/scripts/setupTests.js'],
  snapshotSerializers: ['enzyme-to-json/serializer'],
  moduleDirectories: ['node_modules', 'src'],
};
```

**Comment by theKashey** [comment]
Duplicate of #1252
Your case was fixed in v4.8.8 - just update RHL

**Comment by nryoung** [comment]
@theKashey 	Wow github search really failed me here. I search the closed issues multiple ways and did not find this one.

It works perfectly, thank you so much!

**Comment by theKashey** [comment]
Thank you for creating this issue, I've almost forgot to add final fix for #1252 to the upcoming release.
